index.html

```diff
@@ -49,13 +52,17 @@
     <body>
         <header>
             <h1>Offentligt IT</h1>
-            <h2>Offentlige midler, offentlig software</h2>
+            <h2>Offentlige midler, offentlig IT</h2>
         </header>
         <section class="intro">
             <p>
-                <strong>Offentlig IT</strong> har til formål at forbedre offentlig IT gennem
-                oplysning og debat. Ved at sætte fokus på åbenhed og uafhængighed, viser vi 
-                vejen til en billigere og bedre digital infrastruktur, til gavn for hele Danmark. 
+                <strong>Offentlig IT</strong> er et projekt, der har til
+                formål at åbne og forbedre vores offentlige IT infrastruktur - til gavn for hele Danmark.
+                Vores offentlige IT systemer ejes ikke af Danmark, selvom danskerne har betalt for
+                dem. Vi mener at vi har krav på at eje det vi køber.
+                <br><br>
+                Gennem oplysning og debat viser vi vejen mod bedre og billigere IT. 
```

Comment:
Jeg er stadig ikke fan af denne linje. Det lyder som en organisationsvision. Hvis vi er et projekt må vores mål være konkret. Vi er ikke "in it" for at oplyse og debattere. Vi er "in it" for at ændre udbudsloven.